### PR TITLE
Fixed the wrong vertical alignment of dropdown arrow

### PR DESCRIPTION
**Pull request by petrchutny**
A tiny fix of the dropdown arrow, just moved it 6px to the top.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -87,6 +87,7 @@
     }
   },
   "dependencies": {
+    "@purple/phoenix-components": "^2.5.1",
```
Comment (brno32) [inline package.json:90]: why add this as a dependency?

**Review by brno32 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (petrchutny) [reply package.json:90]: Thanks for asking Alex. I've noticed this being added after I did the "npm i @purple/phoenix-components". I've asked Aleš who said I shouldn't worry about it, but it doesn't seem right to me to modify the package.json in this repo. Maybe you can discard this change from the request?

Comment (brno32) [reply package.json:90]: yeah that's funny. I'm not sure how it added it, but it could definitely lead to some funky behavior when installing the package

**Review by brno32 — APPROVED** [review]
I removed the dependency changes

**Pull request merged**